server/services/clean-up.js

```diff
@@ -169,7 +170,8 @@ const dropStaleOpenParticipantsTable = async () => {
  * @returns {void}
  */
 const invalidateStaleOpenParticipants = async () => {
-  const updateStatement = `DO $$
+  const updateStatement = `START TRANSACTION;
```

Comment:
This is what I was thinking about for transactions 

https://massivejs.org/docs/tasks-and-transactions

So perhaps something like

```
db.withTransaction(async tx => {
  await tx.query(updateStatement);
});

```

Reply:
Oh, okay